.github/workflows/build-custom-docker-image.yml

```diff
@@ -0,0 +1,38 @@
+name: OpenSearch Dashboards Build Custom Docker Image
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Build-Custom-Docker-Image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: append config and custom command
+        env:
+          TRACKING_ID: ${{secrets.GA_TRACKING_ID}}
+          PLUGINS: ${{secrets.PLUGINS}}
+        run: |
+          if [[ $TRACKING_ID ]]; then
+            echo -e "\r\n" >> config/playground/docker/dev/extra_config/osd.yml
+            echo google_analytics_plugin.trackingID: $TRACKING_ID >> config/playground/docker/dev/extra_config/osd.yml
+          fi
+          echo -e "\r\n" >> config/playground/docker/dev/Dockerfile
+          echo CMD [\"python\", \"/opt/opensearch-dashboards/scripts/custom_osd_entry.py\", \"$PLUGINS\"] >> config/playground/docker/dev/Dockerfile
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        env:
+          DOCKER_HUB_REPO: ${{secrets.DOCKER_HUB_REPO}}
+          DOCKER_HUB_TAG: ${{secrets.DOCKER_HUB_TAG}}
```

Comment:
```suggestion
          DOCKER_HUB_TAG: ${{ secrets.DOCKER_HUB_TAG }}
```

Reply:
Thx! Added padding in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/6458665438fb5538564e3fd2a5e5190caeac19a1)